ocaml/xapi-cli-server/cli_frontend.ml

```diff
@@ -3071,6 +3071,15 @@ let rec cmdtable_data : (string * cmd_spec) list =
       ; flags= [Host_selectors]
       }
     )
+  ; ( "host-refresh-server-certificates"
+    , {
+        reqd= []
+      ; optn= []
+      ; help= "Refresh server certificate on host with local host as default"
```

Comment:
'local host as default' is a bit misleading, in the case where xe is not running in dom0

Reply:
I agree; the actual implementation does not have a default. I am also confused by the use of `host` and `host-uuid` parameter. Function `do_op_on_host` evaluates a `host` parameter but other API function use `host-uuid`.

```